code/clue.js

```diff
@@ -40,20 +163,68 @@ const randomSelector = array => {
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+const mystery = {
+  suspects,
+  weapons,
+  rooms:[]
+}
 
+//Killer
 // This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
+
   mystery.killer = randomSelector(suspects)
```

Comment:
So, `suspect` or a `killer`? You haven't initialised your object with property called `killer` so after this line your mystery object has 4 properties : suspects, weapons, rooms and killer. That doesn't make sense